rpms/build.sh

```diff
@@ -43,4 +44,11 @@ $linux rpmbuild --define "_topdir ${rpm_build_dir}" --define "_threads ${threads
         --define "_debugenabled ${debug}" --target ${architecture_target} \
         -ba ${rpm_build_dir}/SPECS/${package_name}.spec
 
+if [[ "${checksum}" == "yes" ]]; then
+    find ${build_dir} -name "*.rpm" -exec bash -c 'sha512sum "$1" > "$1".sha512' bash {} \;
```

Comment:
Change this `if` block as I said in the Debian build.sh script.

Reply:
done in https://github.com/wazuh/wazuh-packages/pull/192/commits/c1caa5e45b8d2b16abc8021bcdead50b47f1447f